python/F3DPythonBindings.cxx

```diff
@@ -224,72 +257,183 @@ PYBIND11_MODULE(pyf3d, module)
+    .def("toggleAnimation", &f3d::interactor::toggleAnimation, "Toggle the animation", "DEPRECATED")
+    .def("startAnimation", &f3d::interactor::startAnimation, "Start the animation", "DEPRECATED")
+    .def("stopAnimation", &f3d::interactor::stopAnimation, "Stop the animation", "DEPRECATED")
+    .def("isPlayingAnimation", &f3d::interactor::isPlayingAnimation,
+      "Returns True if the animation is currently started", "DEPRECATED")
+    .def("enableCameraMovement", &f3d::interactor::enableCameraMovement,
+      "Enable the camera interaction", "DEPRECATED")
+    .def("disableCameraMovement", &f3d::interactor::disableCameraMovement,
+      "Disable the camera interaction", "DEPRECATED")
+    .def("playInteraction", &f3d::interactor::playInteraction, "Play an interaction file",
+      "DEPRECATED")
+    .def("recordInteraction", &f3d::interactor::recordInteraction, "Record an interaction file",
+      "DEPRECATED")
+    .def_static(
+      "getDefaultInteractionsInfo", &f3d::interactor::getDefaultInteractionsInfo, "DEPRECATED");
+
+  loader //
+    .def("hasGeometryReader", &f3d::loader::hasGeometryReader, "DEPRECATED")
+    .def(
+      "loadGeometry", &f3d::loader::loadGeometry, "load geometry to a default scene", "DEPRECATED")
+    .def("hasSceneReader", &f3d::loader::hasSceneReader, "DEPRECATED")
+    .def("loadScene", &f3d::loader::loadScene, "Load a specific full scene file", "DEPRECATED");
+
+  camera //
+    .def("setPosition", &f3d::camera::setPosition, "DEPRECATED: use position property setter")
+    .def("getPosition", py::overload_cast<>(&f3d::camera::getPosition),
+      "DEPRECATED: use position property getter")
+    .def("setFocalPoint", &f3d::camera::setFocalPoint, "DEPRECATED")
+    .def("getFocalPoint", py::overload_cast<>(&f3d::camera::getFocalPoint), "DEPRECATED")
+    .def("setViewUp", &f3d::camera::setViewUp, "DEPRECATED")
+    .def("getViewUp", py::overload_cast<>(&f3d::camera::getViewUp), "DEPRECATED")
+    .def("setViewAngle", &f3d::camera::setViewAngle, "DEPRECATED")
+    .def("getViewAngle", py::overload_cast<>(&f3d::camera::getViewAngle), "DEPRECATED")
+    .def("getViewAngle", py::overload_cast<f3d::angle_deg_t&>(&f3d::camera::getViewAngle),
+      "DEPRECATED")
+    .def("setState", &f3d::camera::setState, "DEPRECATED")
+    .def("getState", py::overload_cast<>(&f3d::camera::getState), "DEPRECATED")
+    .def("getState", py::overload_cast<f3d::camera_state_t&>(&f3d::camera::getState), "DEPRECATED")
+    .def("setCurrentAsDefault", &f3d::camera::setCurrentAsDefault, "DEPRECATED")
+    .def("resetToDefault", &f3d::camera::resetToDefault, "DEPRECATED")
+    .def("resetToBounds", &f3d::camera::resetToBounds, py::arg("zoomFactor") = 0.9, "DEPRECATED");
+
+  window //
+    .def("getType", &f3d::window::getType, "DEPRECATED")
+    .def("getCamera", &f3d::window::getCamera, py::return_value_policy::reference, "DEPRECATED")
+    .def("render", &f3d::window::render, "Render the window", "DEPRECATED")
```

Comment:
this one is duplicated here, should only keep the non-deprecated further up

Reply:
done